R/pkg/DESCRIPTION

```diff
@@ -23,7 +23,7 @@ Suggests:
     testthat,
     e1071,
     survival,
-    arrow (>= 0.15.1)
+    arrow (>= 1.0.0)
```

Comment:
Oh, nice. Are we going to drop Arrow 0.x officially?

Reply:
Yup, I think at least it's okay for SparkR for now.